.github/workflows/mobile-release.yml

```diff
@@ -0,0 +1,65 @@
+name: mobile-release
```

Comment:
only difference between this file and `mobile-build.yml` is the `--auto-submit` flag.